src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -476,6 +479,7 @@ export class EditDataGridPanel extends GridParentComponent {
 		if (newlineMatches && newlineMatches.length > 0) {
 			this.newlinePattern = newlineMatches[0];
 		}
+		// allow-any-unicode-next-line
```

Comment:
Nit: You are not just allowing but replacing unicode next line with null

Reply:
This is telling the hygiene check to specifically ignore the `allow-any-unicode` rule on the next line. That rule flags any code that has unicode symbols in it 

https://github.com/Microsoft/azuredatastudio/blob/8a3d08f0dec1cdf47d7fc442f90fb2227b828375/build/hygiene.js#L44